diff --git a/config-templates/config.php b/config-templates/config.php
index 02acd86..3645014 100644
--- a/config-templates/config.php
+++ b/config-templates/config.php
@@ -62,10 +62,11 @@ $config = [
      * When specified as a relative path, this is relative to the SimpleSAMLphp
      * root directory.
      */
-    'certdir' => 'cert/',
-    'loggingdir' => 'log/',
-    'datadir' => 'data/',
+    'certdir' => '/etc/simplesamlphp/certs/',
+    'loggingdir' => '/var/log/simplesamlphp/',
+    'datadir' => '/var/lib/simplesamlphp/',
     'tempdir' => '/tmp/simplesaml',
+    'attributenamemapdir' => '/etc/simplesamlphp/attributemap/',
 
     /*
      * Some information about the technical persons running this installation.
@@ -292,7 +293,7 @@ $config = [
      *
      */
     'logging.level' => SimpleSAML\Logger::NOTICE,
-    'logging.handler' => 'syslog',
+    'logging.handler' => 'errorlog',
 
     /*
      * Specify the format of the logs. Its use varies depending on the log handler used (for instance, you cannot
@@ -591,7 +592,7 @@ $config = [
      * Options to override the default settings for php sessions.
      */
     'session.phpsession.cookiename' => 'SimpleSAML',
-    'session.phpsession.savepath' => null,
+    'session.phpsession.savepath' => '/var/lib/simplesamlphp/sessions',
     'session.phpsession.httponly' => true,
 
     /*
@@ -712,12 +713,14 @@ $config = [
      *     ],
      * ],
      *
+     * or:
+     * 'memcache_store.servers' => [
+     *     [
+     *         ['hostname' => 'localhost'],
+     *     ],
+     * ],
      */
-    'memcache_store.servers' => [
-        [
-            ['hostname' => 'localhost'],
-        ],
-    ],
+    'memcache_store.servers' => [],
 
     /*
      * This value allows you to set a prefix for memcache-keys. The default
@@ -727,7 +730,7 @@ $config = [
      * than one instance is using memcache, you probably want to assign
      * a unique value per instance to this setting to avoid data collision.
      */
-    'memcache_store.prefix' => '',
+    'memcache_store.prefix' => 'SimpleSAMLphp',
 
     /*
      * This value is the duration data should be stored in memcache. Data
@@ -1066,7 +1069,7 @@ $config = [
      * This option allows you to specify a directory for your metadata outside of the standard metadata directory
      * included in the standard distribution of the software.
      */
-    'metadatadir' => 'metadata',
+    'metadatadir' => '/etc/simplesamlphp/metadata/',
 
     /*
      * This option configures the metadata sources. The metadata sources is given as an array with