diff --git a/config-templates/config.php b/config-templates/config.php
index c0bfdd4..e0404f9 100644
--- a/config-templates/config.php
+++ b/config-templates/config.php
@@ -67,10 +67,11 @@ $config = [
      * When specified as a relative path, this is relative to the SimpleSAMLphp
      * root directory.
      */
-    'cachedir' => '/var/cache/simplesamlphp',
-    //'loggingdir' => '/var/log/',
-    //'datadir' => '/var/data/',
-    //'tempdir' => '/tmp/simplesamlphp',
+    'cachedir' => '/var/cache/simplesamlphp/',
+    'loggingdir' => '/var/log/simplesamlphp/',
+    'datadir' => '/var/lib/simplesamlphp/',
+    'tempdir' => '/tmp/simplesamlphp',
+    'attributenamemapdir' => '/etc/simplesamlphp/attributemap/',
 
     /*
      * Certificate and key material can be loaded from different possible
@@ -90,7 +91,7 @@ $config = [
      * directory. Note that locations with no prefix included will be treated
      * as file locations.
      */
-    'certdir' => 'cert/',
+    'certdir' => '/etc/simplesamlphp/certs/',
 
     /* To load a certificate or key from the database, it should be specified
      * as 'pdo://<id>' where <id> is the identifier in the database table that
@@ -366,7 +367,7 @@ $config = [
      * loggingdir above to 'null'.
      */
     'logging.level' => SimpleSAML\Logger::NOTICE,
-    'logging.handler' => 'syslog',
+    'logging.handler' => 'errorlog',
 
     /*
      * Specify the format of the logs. Its use varies depending on the log handler used (for instance, you cannot
@@ -560,7 +561,8 @@ $config = [
         'exampleauth' => false,
         'core' => true,
         'admin' => true,
-        'saml' => true
+        'saml' => true,
+        'metarefresh' => true
     ],
 
 
@@ -655,7 +657,7 @@ $config = [
      * Options to override the default settings for php sessions.
      */
     'session.phpsession.cookiename' => 'SimpleSAML',
-    'session.phpsession.savepath' => null,
+    'session.phpsession.savepath' => '/var/lib/simplesamlphp/sessions',
     'session.phpsession.httponly' => true,
 
     /*
@@ -776,12 +778,14 @@ $config = [
      *     ],
      * ],
      *
+     * or:
+     * 'memcache_store.servers' => [
+     *     [
+     *         ['hostname' => 'localhost'],
+     *     ],
+     * ],
      */
-    'memcache_store.servers' => [
-        [
-            ['hostname' => 'localhost'],
-        ],
-    ],
+    'memcache_store.servers' => [],
 
     /*
      * This value allows you to set a prefix for memcache-keys. The default
@@ -791,7 +795,7 @@ $config = [
      * than one instance is using memcache, you probably want to assign
      * a unique value per instance to this setting to avoid data collision.
      */
-    'memcache_store.prefix' => '',
+    'memcache_store.prefix' => 'SimpleSAMLphp',
 
     /*
      * This value is the duration data should be stored in memcache. Data
@@ -1065,7 +1069,7 @@ $config = [
      * This option allows you to specify a directory for your metadata outside of the standard metadata directory
      * included in the standard distribution of the software.
      */
-    'metadatadir' => 'metadata',
+    'metadatadir' => '/etc/simplesamlphp/metadata/',
 
     /*
      * This option configures the metadata sources. The metadata sources is given as an array with
